### PR TITLE
LP-1389 custom settings save issue resolved

### PR DESCRIPTION
**Pull request by zeshanarifios**
**How to test?**

Open studio shell by running `make studio-shell` in devstack directory in terminal.

**vim cms.env.json**
```
"NODEBB_ENDPOINT": "http://your-system-ip-address:4567"
```
**vim cms.auth.json**
Add
```
"MANDRILL_API_KEY": "MANDRILL_API_KEY here",
"MAILCHIMP_API_KEY": "MAILCHIMP_API_KEY here",
```

### CODE REVIEW

File: cms/envs/production.py [hunk 1]
```diff
@@ -617,6 +617,19 @@
 # We continue to load production.py over aws.py
 plugin_settings.add_plugins(__name__, plugin_constants.ProjectType.CMS, plugin_constants.SettingsType.PRODUCTION)
 
+########################## Philu related settings ########################
```
Comment (ahmed-faraz46) [inline cms/envs/production.py]: We've all these setting in `aws.py` would you please elaborate why those settings are not working now. Also if those are no more required please remove them from `aws.py`

File: common/djangoapps/custom_settings/helpers.py [hunk 2]
```diff
@@ -31,7 +31,8 @@ def validate_course_open_date(settings, course_open_date):
             course_open_date = course_open_date.replace(tzinfo=utc)
             course = CourseOverview.objects.get(id=settings.id)
 
-            if course.end < course_open_date or course_open_date < course.start:
+            if (course.end and course.end < course_open_date) or \
```
Comment (ahmed-faraz46) [inline common/djangoapps/custom_settings/helpers.py]: I'm wondering why how this sanity was working in ficus. Would you please revert this change and verify again because I guess when you modified this sanity `CourseOverview` object was not being saved at the first place now we have fixed that, I believe it would work now.

File: cms/envs/production.py (hunk 1 above)
Comment (zeshanarifios) [reply cms/envs/production.py]: if you open `aws.py` you can see this warning `logging.warn('DEPRECATION WARNING: aws.py has been deprecated, you should use production.py instead.')` That's why I put these settings in `production.py`.
